src/apiClient.js

```diff
@@ -426,6 +426,22 @@ ApiClient_.prototype.fetchMiningPoolsDistribution = function (slug, from, to) {
   return this.conn.graphQLQuery(query, 'miningPoolsDistribution')
 }
 
+ApiClient_.prototype.fetchMinersBalance = function (slug, from, to) {
+  var query = {
+    'query': '{\
+       minersBalance(slug: "' + slug + '",\
+                     from: "' + toUTC_(from) + '",\
+                     to: "' + toUTC_(to) + '",\
+                     interval: "1d") {\
```

Comment:
should this be a default value, but changeable?

Reply:
You mean to add interval argument to the functions with a default value of "1d"?